torch/fx/graph.py

```diff
@@ -588,12 +597,27 @@ def emit_node(node : Node):
 
         code = ''.join(body).lstrip('\n')
         code = '\n'.join('    ' + line for line in code.split('\n'))
+
+        # remove counter and generate lineno to node index mapping
```

Comment:
It's not clear to me why you need to remove the counter lol

Reply:
To keep the GraphModule code clean